miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultKitDescriptorService.java

```diff
@@ -161,4 +165,38 @@ public long count(Consumer<String> errorHandler, PaginationFilter... filter) thr
     return kitStore.count(errorHandler, filter);
   }
 
+  @Override
+  public DeletionStore getDeletionStore() {
+    return deletionStore;
+  }
+
+  @Override
+  public AuthorizationManager getAuthorizationManager() {
+    return authorizationManager;
+  }
+
+  @Override
+  public ValidationResult validateDeletion(KitDescriptor object) throws IOException {
+    ValidationResult result = new ValidationResult();
+    switch (object.getKitType()) {
```

Comment:
Does it make sense to do all the checks in case there's some database shenanigans? Worst case, they'll be zero.